From 788911c779fbf2ec7445309e25b465b599ffcf77 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Tue, 16 Nov 2021 14:43:21 +0100
Subject: [PATCH 272/529] ARM: dts: stm32: add ltdc support to stm32mp13x soc
 family

Add ltdc support to STM32MP13x SoC family.

Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Change-Id: I7e5d99d9b50211ae80947288671477b5d28d9036
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/228337
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/280102
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
---
 arch/arm/boot/dts/stm32mp135.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

--- a/arch/arm/boot/dts/stm32mp135.dtsi
+++ b/arch/arm/boot/dts/stm32mp135.dtsi
@@ -8,5 +8,15 @@
 
 / {
 	soc {
+		ltdc: display-controller@5a001000 {
+			compatible = "st,stm32-ltdc";
+			reg = <0x5a001000 0x400>;
+			interrupts = <GIC_SPI 88 IRQ_TYPE_LEVEL_HIGH>,
+				     <GIC_SPI 89 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&rcc LTDC_PX>;
+			clock-names = "lcd";
+			resets = <&scmi_reset RST_SCMI_LTDC>;
+			status = "disabled";
+		};
 	};
 };
